runelite-api/src/main/java/net/runelite/api/Varbits.java

```diff
@@ -304,6 +304,8 @@
 	PERSONAL_POINTS(5422),
 	RAID_PARTY_SIZE(5424),
 
+	RAID_HEALTH_BAR_OVERLAY_VALUE(6099),
```

Comment:
What value does this hold? Seems like it's an int of the current health of the monster? Add a comment to describe this.

Reply:
I've added another varbit for the max health of a monster, and a comment to describe these varbits